packages/components/link.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { MouseEvent } from "react";
+import connect, { useConnect } from "@frontity/connect";
```

Comment:
I'm not sure if it's expected but when I was importing `connect` from `frontity` package I was getting typescript errors saying connect only expects 1 argument.

Reply:
That was fixed in a beta version, but not sure if it ended in the release. Are you using the last version of the `frontity` package?

Reply:
That problem comes from this declaration https://github.com/frontity/frontity/blob/8dcedad029fedbb0cfb0799d15a1f7abb924b1d4/packages/types/src/connect.ts#L13-L20

It was fixed in a beta version for the infinite scroll hook, but maybe it never made it to the release. I don't know how the move from beta to release is being done.

This is how it should look like, in case it helps
```ts
export interface ConnectFunction {
  <Comp extends FunctionComponent<any>>(comp: Comp): FunctionComponent<
    FilterInjectedProps<ComponentProps<Comp>>
  >;
  <Comp extends FunctionComponent<any>>(
    comp: Comp,
    options: { injectProps: boolean }
  ): FunctionComponent<FilterInjectedProps<ComponentProps<Comp>>>;
  <Comp extends ComponentClass<any>>(comp: Comp): ComponentClass<
    FilterInjectedProps<ComponentProps<Comp>>
  >;
}
```

You can implement the change to fix the issue and just create two changesets and merge both with the same PR, I think, but @luisherranz will know better :)

Reply:
Our fault. Thanks for the report guys. I've made a quick fix here: https://github.com/frontity/frontity/pull/508. Once it's merged we can update this PR and import it again from `frontity` instead of `@frontity/connect`.

This duplication of types between `@frontity/connect` and `frontity` is something I want to solve as part of our migration to the new version of `react-easy-state` in https://github.com/frontity/frontity/pull/415.